added description of how to save layers without compression (#4832)
Napari/napari: docs/howtos/layers/image.md

Currently if you pass contrast limits as a keyword argument to a layer then full
extent of the contrast limits range slider will be set to those values.

## Saving without image compression

> When saving a labels layer, lossless zlib compression is applied by default. 

Reviewer: ```suggestion
When saving an image layer, lossless zlib compression is applied by default. 
```
